chainstate/src/detail/tests/events_tests.rs

```diff
@@ -61,14 +61,15 @@ fn simple_subscribe() {
     });
 }
 
+proptest! {
```

Comment:
I don't like formatting with braces that skips indentation... it's confusing... I lived hell in C++ with namespace that doesn't indent. Should we change this in our format settings file?